packages/veto-frontend/src/types/axios.d.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios'
+
```

Comment:
Why is this needed? 
Axios already has type definition https://github.com/axios/axios/blob/master/index.d.ts